src/helpers/strings.js

```diff
@@ -120,5 +119,109 @@ export default {
         });
 
         return (params.length > 0) ? _vsprintf(format, params) : format;
+    },
+
+    /**
+     * Changes the string to lowercase.
+     * Example usage:
+     * 		{{lowercase 'JUST WOW!!!'}}   => 'just wow!!!'
+     *
+     * @param  string param
+     * @return string
+     */
+    lowercase: (param) => {
+        return isString(param) ? param.toLowerCase() : param;
+    },
+
+    /**
+     * Changes the string to uppercase.
+     * Example usage:
+     * 		{{uppercase 'just wow!!!'}}   => 'JUST WOW!!!'
+     *
+     * @param  string param
+     * @return string
+     */
+    uppercase: (param) => {
+        return isString(param) ? param.toUpperCase() : param;
+    },
+
+    /**
+     * Get the first element of a collection/array.
+     * Example usage:
+     * 		{{first ['David', 'Miller', 'Jones']}}   => 'David'
```

Comment:
handlebars don't support expressions like this `{{first ['David', 'Miller', 'Jones']}}` it should be like this `{{first someArray}}`